sdk/textanalytics/Azure.AI.TextAnalytics/src/CategorizedEntity.cs

```diff
@@ -67,5 +72,10 @@ internal CategorizedEntity(string text, string category, string subcategory, dou
         /// Gets the length of the matching text in the input document.
         /// </summary>
         public int Length { get; }
+
+        /// <summary>
+        /// Gets the collection of entity resolutions.
```

Comment:
Feel free to add this aka link to the docs: https://aka.ms/azsdk/language/ner-resolutions

Reply:
Thank you!